TACC/Core-Portal
# task/FP-1131: Add Member role to shared workspace (frontend)

## Overview: ##
Frontend portion of FP-1131.
## Related Jira tickets: ##

* [FP-1131](https://jira.tacc.utexas.edu/browse/FP-1131)

## Summary of Changes: ##
- Replace the "owner/editor" frontend roles with roles that are explicitly tied to Tapis system roles (OWNER/ADMIN/USER/GUEST). 

## Testing Steps: ##
1. Navigate to a project with multiple users, which you are an admin on.
2. Try changing the system role for other users on the project.
3. Add a user to a project and check that their role is USER.

## UI Photos:
<img width="746" alt="image" src="https://user-images.githubusercontent.com/12601812/163420532-3afe1288-07f4-4729-aeb2-47ba950d9f56.png">


## Notes: ##
